src/modules/homepage/index.tsx

```diff
@@ -0,0 +1,430 @@
+
+const StyledBottomNavigationContainer = styled('div')({
+  position: 'sticky',
+  bottom: 0,
+  left: 0,
+  right: 0,
+  width: '375px',
+  zIndex: '1000',
+  backgroundColor: '#fff',
+});
+
+const StyledRectangle = styled(Box)({
+  textAlign: 'center',
+  width: '375px',
+  height: '312px',
+  gap: '0px',
+  borderRadius: '0px 0px 32px 32px',
+  background:
+    'linear-gradient(183.33deg, #FFF6E6 9.56%, rgba(248, 237, 216, 0) 124.27%)',
+});
+
+export const Homepage = () => {
+  return (
+    <Stack sx={{ minHeight: '100vh', position: 'relative', width: '375px' }}>
+      <StyledRectangle>
+        <Box
+          display='flex'
+          justifyContent='space-between'
+          alignItems='center'
+          maxWidth='375px'
+          height='44px'
+          padding='0 16px'
+        >
+          <Typography variant='body2' fontWeight='700'>
+            <DateTime />
+          </Typography>
+          <Box display='flex' alignItems='center' gap='4px'>
+            <SignalCellularAltIcon fontSize='small' />
+            <WifiIcon fontSize='small' />
+            <BatteryFullIcon
+              fontSize='small'
+              sx={{ transform: 'rotate(90deg)' }}
+            />
+          </Box>
+        </Box>
+
+        <Box
+          display='flex'
+          justifyContent='space-between'
+          alignItems='center'
+          maxWidth='375px'
+          height='64px'
+          padding='0 16px'
+        >
+          <Box
+            sx={{
+              border: '2px solid #7F3DFF',
+              borderRadius: '50%',
+              padding: '2px',
+            }}
+          >
+            <Avatar alt='user image' src={personImage} />
+          </Box>
+
+          <Box display='flex' flexDirection='row' alignItems='center'>
+            <IconButton>
+              <ArrowDropDown />
+            </IconButton>
+            <Typography>October</Typography>
+          </Box>
+
+          <Box sx={{ color: '#7F3DFF' }}>
+            <NotificationsIcon />
+          </Box>
+        </Box>
+
+        <Typography sx={{ color: '#91919F', fontSize: '14px' }}>
+          Account Balance
+        </Typography>
+        <Typography
+          sx={{ color: '#161719', fontSize: '40px', fontWeight: '600' }}
+        >
+          $9400
+        </Typography>
+
+        <Box display='flex' flexDirection='row' justifyContent='space-between'>
+          <Box
+            width='164px'
+            height='80px'
+            borderRadius='28px'
+            bgcolor='#00A86B'
+            marginLeft='16px'
+            display='flex'
+            flexDirection='row'
+            alignItems='center'
+          >
+            <Box
+              width='44px'
+              height='44px'
+              borderRadius='16px'
+              bgcolor='#fff'
+              color='#00A86B'
+              display='flex'
+              flexDirection='column'
+              alignItems='center'
+              justifyContent='center'
+              marginLeft='16px'
```

Comment:
Remove margin, add padding instead to the entire "green"/"red" element so it's consistent on all sides. For the space between icon and text, use flex gap.

Reply:
fixed